src/dotnet/commands/dotnet-test/LocalizableStrings.resx

```diff
@@ -162,7 +162,9 @@
   </data>
   <data name="CmdLoggerDescription" xml:space="preserve">
     <value>Specify a logger for test results.
-                                        Example: --logger "trx[;LogFileName=&lt;Defaults to unique file name&gt;]"
+                                        Examples:
+                                        Log in trx format using a unqiue file name: --logger trx
+                                        Log in trx format using the specified file name: --logger "trx;LogFileName=&lt;TestResults.resx&gt;"
```

Comment:
filename should be TestResults.trx
`Log in trx format using the specified file name: --logger "trx;LogFileName=&lt;TestResults.trx&gt;"`

Reply:
Oh wow... 🤦‍♂️ , Fixed it #7557